src/java.base/share/classes/java/io/File.java

```diff
@@ -1813,18 +1813,26 @@ public boolean canExecute() {
      * <p> A particular Java platform may support zero or more
      * hierarchically-organized file systems.  Each file system has a
      * {@code root} directory from which all other files in that file system
-     * can be reached.  Windows platforms, for example, have a root directory
-     * for each active drive; UNIX platforms have a single root directory,
-     * namely {@code "/"}.  The set of available filesystem roots is affected
-     * by various system-level operations such as the insertion or ejection of
-     * removable media and the disconnecting or unmounting of physical or
-     * virtual disk drives.
+     * can be reached.
      *
      * <p> This method returns an array of {@code File} objects that denote the
      * root directories of the available filesystem roots.  It is guaranteed
      * that the canonical pathname of any file physically present on the local
      * machine will begin with one of the roots returned by this method.
```

Comment:
@Nagata-Haruhito Per @AlanBateman 's suggestion, would you please add this sentence

“There is no guarantee that the root directory can be accessed”

or the equivalent here after “returned by this method” and after the sentence that begins with “A file system ...” in the specification of `java.nio.file.FileSystem::getRootDirectories` and so update this PR?

Thanks.

Reply:
Thanks for your advice. I changed description as your advice. I will publish CSR with this change.